tests/helpers.js

```diff
@@ -161,8 +161,7 @@ export function getRegistryAddressFromBrowser () {
 }
 
 export function resetDb () {
-  Meteor.users.remove({ 'profile.name': 'Guildenstern' })
-  Meteor.users.remove({ 'emails.address': 'guildenstern@rosencrantz.com' })
+  Meteor.users.remove({})
```

Comment:
Does anyone see an issue with removing all users after/before each test? This is to ensure that users will be removed even after email or name is changed (which is a scenario we need to test)

Reply:
it no tests break, your safe :-)